programs/migrations/0131_rename_cesn_program_abbreviations.py

```
@@ -0,0 +1,68 @@
+# Generated by Django 4.2.22 on 2026-01-28 03:53
+
+from django.db import migrations
+
+
+def rename_co_energy_calculator_to_cesn(apps, schema_editor):
```

Ruff (0.14.14)

[warning] Unused function argument: `schema_editor`

(ARG001)